documentation/samplefiles-dpg/readme.java.md

```diff
@@ -0,0 +1,17 @@
+## Java
+
+- If `[[PackageName]]` is `azure-service-database`, then `[[Namespace]]` shall be `azure.service.database`, see [Namespace](https://azure.github.io/azure-sdk/java_introduction.html#namespaces).
+- `[[ServiceName]]`, see [Service Client](https://azure.github.io/azure-sdk/java_introduction.html#service-client).
+
+`[[PackageName]]` and `[[Namespace]]` is required to be reviewed by [SDK arch board](https://github.com/Azure/azure-sdk/issues).
+
+``` yaml $(java)
+output-folder: $(java-sdks-folder)/sdk/[[ServiceFolder]]/[[PackageName]]]
```

Comment:
Do we have the doc to describe all the supported properties to be set for each SDK？ If so, please add the link of that doc in the sample file.  
@msyyc ,  @dw511214992 , @weidongxu-microsoft

Reply:
python added